packages/manager/modules/telecom-spare/src/index.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Difference between loading a module and importing a module
+ *
+ * Loading a module means:
+ * - loading its content
+ * - executing its content
+ *
+ * Modules are singletons, so only one instance of them exists
+ *
+ * Loading a module may have the following consequences:
+ * - if the content exports a feature, then it will be available to be imported by other modules
+ * - if the content changes the global object, then the changes will be done
+ *
+ * Importing is an optional step that follows loading
+ *
+ * It means, using in the importing module a feature exposed by the exporting module
+ */
+
+// here we import the feature exported by the "angular" module and we call it "angular"
+import angular from 'angular';
+
+/**
+ * Here we load the "@ovh-ux/manager-core" module
+ * The body of "@ovh-ux/manager-core" executes code on the "angular" object
+ * As "angular" is a singleton, what "@ovh-ux/manager-core" executes on it
+ * is reflected here
+ */
+import '@ovh-ux/manager-core';
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
```

Comment:
```suggestion
import '@ovh-ux/ng-ovh-telecom-universe-components';
import '@ovh-ux/ng-ovh-mondial-relay';
```

And use moduleName strings in module declaration.

Reply:
but it's done like mine in the telecom manager app? I don't understand why you want I'm doing like that.

Reply:
The difference between the app is that you're in a module so you don't have a dependency but a peer dependency. This can cause issue because peer deps are not present at the build and are brought by host